src/library/Uno.Material/Styles/Controls/v2/ToggleButton.xaml

```diff
@@ -495,29 +492,31 @@
 								</VisualState>
 								<VisualState x:Name="Disabled">
 									<VisualState.Setters>
-										<Setter Target="ContentPresenter.Visibility" Value="Visible" />
+										<Setter Target="ContentPresenter.Opacity" Value="1" />
 										<Setter Target="ContentPresenter.Foreground" Value="{ThemeResource IconToggleButtonForegroundDisabled}" />
-										<Setter Target="AlternateContentPresenter.Visibility" Value="Collapsed" />
+										<Setter Target="AlternateContentPresenter.Opacity" Value="0" />
 										<Setter Target="AlternateContentPresenter.Foreground" Value="{ThemeResource IconToggleButtonForegroundDisabled}" />
 										<Setter Target="RootGrid.Background" Value="{ThemeResource IconToggleButtonBackgroundDisabled}" />
 										<Setter Target="RootGrid.BorderBrush" Value="{ThemeResource IconToggleButtonBorderBrushDisabled}" />
 									</VisualState.Setters>
 								</VisualState>
+								<!-- For the checked states, we need to also validate if AlternateContent is set first before changing opacity -->
+								<!-- https://github.com/unoplatform/Uno.Themes/issues/1132 -->
```

Comment:
is the converter truly necessary?
my impression is that having " "(single space) are alt-content was enough to prevent the crash

Reply:
@kazo0 what do you think regarding the latest changes ?

Reply:
@Xiaoy312 Yes, we need the converter. Without it, the AlternateContent won't show up for the Checked state when IT IS provided.

@kazo0 If you're ok with the changes, I'll resolve this convo and we can merge this.